src/_samples/index.md

```diff
@@ -226,11 +240,12 @@ class Orbiter extends Spacecraft with Manned {
 
 ## Interfaces
 
-Dart has no `interface` keyword. Instead, all classes implicitly define an interface. Therefore, you can `implement` any class.
+Dart has no `interface` keyword. Instead, all classes implicitly define an interface. Therefore, you can implement any class.
 
+<?code-excerpt "lib/samples/spacecraft.dart (implements)"?>
 {% prettify dart %}
 class MockSpaceship implements Spacecraft {
-  // ...
+  // ···
```

Comment:
I just noticed the changes from `...` to `···`. It looks strange to me because of the vertical alignment—like you're showing single-quote characters, rather than  an ellipsis (which I expect to look like [or be] three periods).

Reply:
Wow, you are right. Ok, I'll have to make a sweeping change of these.

Reply:
I've reverted to using dots in the samples code. I'll address other instances separately.